lib/split/dashboard/views/index.erb

```diff
@@ -24,3 +26,16 @@
   <p class="intro">No experiments have started yet, you need to define them in your code and introduce them to your users.</p>
   <p class="intro">Check out the <a href='https://github.com/splitrb/split#readme'>Readme</a> for more help getting started.</p>
 <% end %>
+
+<div class="dashboard-controls dashboard-controls-bottom">
+  <form action="<%= url "/initialize_experiment" %>" method='post'>
+    <label>Add unregistered experiment: </label>
+    <select name="experiment" id="experiment-select">
+      <option selected disabled>experiment</option>
+      <% @unintialized_experiments.sort.each do |experiment_name| %>
+          <option value="<%= experiment_name %>"><%= experiment_name %></option>
+      <% end %>
+    </select>
+    <input type="submit" id="register-experiment-btn" value="register experiment"/>
+  </form>
+<div>
```

Comment:
Already in our code: https://github.com/clio/split/blob/master/lib/split/dashboard/views/index.erb#L30-L41